extensions/VisualEditor/wikia/modules/ve/dm/ve.dm.WikiaTemplateModel.js

```diff
@@ -6,6 +6,8 @@
 ve.dm.WikiaTemplateModel = function VeDmWikiaTemplateModel( transclusion, target ) {
 	// Parent constructor
 	ve.dm.WikiaTemplateModel.super.call( this, transclusion, target );
+
+	this.spec = transclusion.isInfobox ? new ve.dm.WikiaInfoboxSpecModel( this ) : new ve.dm.MWTemplateSpecModel( this );
```

Comment:
Not really.
Btw. in VisualEditor code we do not access properties of other objects directly - we use getters (so you should create getIsInfobox and use it).
There are few exceptions: object.$, object.$element and when accessing properties of third party objects (for instance anchorOffset and anchorNode of browser's selection).